### PR TITLE
Update url-examples.md

### PR DESCRIPTION
**Pull request by YuriKravetc**
+additional examples

### CODE REVIEW

File: docs/url-examples.md [hunk 1]
```diff
@@ -139,6 +139,195 @@ Additional information:
 * [How to import CSV data](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-csv-data)
 
 
+## /api/v1/labels
+
+**Get a list of label names
```
Comment (hagen1778) [inline docs/url-examples.md:144]: `**` is this a typo?

File: docs/url-examples.md [hunk 2]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+Cluster:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://<vmselect>:8481/select/0/prometheus/api/v1/labels'
+```
+
+</div>
+
+**Querying label values
```
Comment (hagen1778) [inline docs/url-examples.md:164]: ** is this a typo?

File: docs/url-examples.md [hunk 3]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+</div>
+
+Additional information: 
+* [Prometheus querying API usage](https://docs.victoriametrics.com/#prometheus-querying-api-usage)
+* [Querying label values](https://prometheus.io/docs/prometheus/latest/querying/api/#querying-label-values)
+* [Getting label names](https://prometheus.io/docs/prometheus/latest/querying/api/#getting-label-names)
+
+## /api/v1/query
+
+**Performs PromQL instant query
```
Comment (hagen1778) [inline docs/url-examples.md:191]: it is not only PromQL, but MetricsQL as well. 
The description also doesn't explain what `instant` means. Also, no all extra params are mentioned.
Please take https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries as reference.

File: docs/url-examples.md [hunk 4]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+
+Cluster:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://<vmselect>:8481/select/0/prometheus/api/v1/query?query=vm_http_request_errors_total'
+```
+
+</div>
+
```
Comment (hagen1778) [inline docs/url-examples.md:210]: Would be also nice to show example with complex query which needs encoding. For example, how to execute the following query sum(increase(foo{status="bar"}[5m])) by (status) ?

File: docs/url-examples.md [hunk 5]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+Additional information: 
+* [Prometheus querying API usage](https://docs.victoriametrics.com/#prometheus-querying-api-usage)
+* [Instant queries](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries)
+
+## /api/v1/query_range
+
+
```
Comment (hagen1778) [inline docs/url-examples.md:217]: nit: extra line

File: docs/url-examples.md [hunk 6]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+**Performs PromQL range_query
```
Comment (hagen1778) [inline docs/url-examples.md:218]: It doesn't explain what is `range_query`

File: docs/url-examples.md [hunk 7]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+
+Single:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://localhost:8428/prometheus/api/v1/query_range?query=vm_http_request_errors_total&start=2021-02-22T19:10:30.781Z&step=20m'
```
Comment (hagen1778) [inline docs/url-examples.md:224]: would be nice to list accepted params and their defaults if omitted

File: docs/url-examples.md [hunk 8]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+</div>
+
+Additional information: 
+* [Prometheus querying API usage](https://docs.victoriametrics.com/#prometheus-querying-api-usage)
+* [Finding series by label matchers](https://prometheus.io/docs/prometheus/latest/querying/api/#finding-series-by-label-matchers)
+
+## /api/v1/status/tsdb
+
+**Cardinality statistics 
+
```
Comment (hagen1778) [inline docs/url-examples.md:280]: nit: extra line

**Review by hagen1778 — CHANGES_REQUESTED** [review]

File: docs/url-examples.md [hunk 9]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+## /api/v1/query
+
+**Performs PromQL/MetricsQL instant query**
+
+Single:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://localhost:8428/prometheus/api/v1/query?query=vm_http_request_errors_total&start=2021-02-22T19:10:30.781Z'
```
Comment (valyala) [inline docs/url-examples.md:197]: The `/api/v1/query` doesn't accept `start` query arg - see https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries

File: docs/url-examples.md [hunk 10]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+
+Cluster:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://<vmselect>:8481/select/0/prometheus/api/v1/labels'
+```
+
+</div>
+
```
Comment (valyala) [inline docs/url-examples.md:163]: Missing links with additional information about `/api/v1/labels` similar to `/api/v1/label/<labelName>/values` below

File: docs/url-examples.md (hunk 4 above)
Comment (valyala) [reply docs/url-examples.md:210]: Agreed

File: docs/url-examples.md [hunk 11]
```diff
@@ -139,6 +139,195 @@ Additional information:
...
+**Querying label values**
```
Comment (valyala) [inline docs/url-examples.md:164]: Missing `## /api/v1/label/<label_name>/values` line

**Review by valyala — CHANGES_REQUESTED** [review]

File: docs/url-examples.md (hunk 9 above)
Comment (YuriKravetc) [reply docs/url-examples.md]: Changed to "time"

File: docs/url-examples.md (hunk 3 above)
Comment (YuriKravetc) [reply docs/url-examples.md]: Added a link to additional info with the mentioned explanation + added MetricsQL part

File: docs/url-examples.md (hunk 4 above)
Comment (YuriKravetc) [reply docs/url-examples.md:218]: Added, thanks for your assistance!

**Comment by valyala** [comment]
Thanks!

**Pull request merged**